site/docs/v1/tech/apis/jwt.adoc

```diff
@@ -542,3 +548,55 @@ The decoded JWT payload. The payload contains the identity claims for the user.
 ----
 include::docs/src/json/jwt/validate-response.json[]
 ----
+
+== Vend a JWT
+
+This API is used to create a free-form access token (JWT) with claims defined by the caller. The only "reserved" claims that cannot be specified are `exp`, `iat`. The `iat` the issued at time of the JWT and the `exp` the expiration time of the JWT as computed by adding either the user passed, or the tenant [field]#timeToLiveInSeconds# to the `iat`. If passed into the [field]#claims# object these claims will be ignored.
+
+=== Request
+
+[.api-authentication]
+link:/docs/v1/tech/apis/authentication#api-key-authentication[icon:lock[type=fas]] Vend a JWT
+[.endpoint]
+.URI
+--
+[method]#POST# [uri]#/api/jwt/vend#
+--
+
+include::docs/v1/tech/apis/_x-fusionauth-tenant-id-header-scoped-operation.adoc[]
+
+==== Request Body
+
+[.api]
+[field]#claims# [type]#[Object]# [optional]#Optional#::
+An object of claims to apply to (associate with?) this JWT.  If any of the three "reserved" claims `exp`, `iat`, or `jti` are specified they will be ignored.
```

Comment:
```suggestion
An set of claims to add to this JWT.  If any of the "reserved" claims, `exp` or `iat`, are specified they will be ignored. Otherwise, the claims can be any valid JSON value.
```